msprime/intervals.py

```diff
@@ -82,34 +159,70 @@ def size(self):
     @property
     def mean_rate(self):
         """
-        Return the weighted mean across all windows of the entire map.
+        The mean rate over this map, from map.start_position to map.end_position.
```

Comment:
Worth noting here the semantics of what start and end mean, or maybe just giving a pointer back to the contructor docs.

Reply:
We should be a bit more precise here, too. It's the mean rate weighted by genome span, right? Someone might think this was just the unweighted mean of the different values.